### PR TITLE
Paperwork update

### PR DESCRIPTION
**Pull request by symphorien**
See https://github.com/NixOS/nixpkgs/pull/81921, which I closed involuntarily.

### CODE REVIEW

**Comment by aszlig** [comment]
@symphorien: Currently on the road, will review tomorrow. Thanks for the pull request.

File: pkgs/applications/office/paperwork/default.nix [hunk 1]
```diff
@@ -46,9 +54,27 @@ python3Packages.buildPythonApplication rec {
...
 
+  postInstall = ''
+    # paperwork-shell needs to be re-wrapped with access to paperwork
+    cp ${python3Packages.paperwork-backend}/bin/.paperwork-shell-wrapped $out/bin/paperwork-shell
+    # install desktop files and icons
+    # paperwork-shell install copies everything to $HOME/.local/share instead of $out/share
+    mkdir -p home/.local
+    mkdir -p $out/share
+    ln -s $out/share home/.local/share
+    HOME=$(readlink -f home) $out/bin/paperwork-shell install
```
Comment (aszlig) [inline pkgs/applications/office/paperwork/default.nix]: What about the following instead?

```suggestion
    XDG_DATA_HOME="$out/share" "$out/bin/payperwork-shell" install
```

Also, did you check whether payperwork-shell actually works at runtime?

Comment (symphorien) [reply pkgs/applications/office/paperwork/default.nix]: yes your suggestion works :)
(minus the typo)

about paperwork-shell: yes, for example `result/bin/paperwork-shell chkdeps paperwork` works whereas it used to complain about a missing paperwork module.

**Comment by symphorien** [comment]
And now that I once again failed at git, I have to close this pull request and reopen a new one. Sorry for the inconvenience.

**Pull request closed without merging**